GsaGH/Components/2_Geometry/Edit1dMember.cs

```diff
@@ -268,12 +268,12 @@ protected override void SolveInstance(IGH_DataAccess da) {
 
       GsaBool6Goo start = null;
       if (da.GetData(8, ref start)) {
-        mem.ReleaseStart = start.Value;
+        mem.ReleaseStart = start.Value.Negate();
       }
 
       GsaBool6Goo end = null;
       if (da.GetData(9, ref end)) {
-        mem.ReleaseEnd = end.Value;
+        mem.ReleaseEnd = start.Value.Negate();
```

Comment:
you mean `end`

Reply:
a bit confusing, we negate the input, so that we can negate it again while we set the values?

Reply:
We negate input to pass correct value to Gsa Member. We negate output to display information correctly.